bin/installMaven

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# download and unzip Maven to ~/maven directory
+# and overwrite the mvn command to run this Maven version instead of Travis' standard maven version (3.5.2)
+
+set -euo pipefail
+
+echo "Setup Maven"
+mkdir -p ~/maven
+pushd ~/maven > /dev/null
+if [ ! -d "apache-maven-$1" ]; then
+  echo "Download Maven $1"
+  wget --quiet --continue  "https://archive.apache.org/dist/maven/maven-3/$1/binaries/apache-maven-$1-bin.zip" 
+  unzip apache-maven-$1-bin.zip -d ~/maven> /dev/null
```

Comment:
The `$1` in the argument of `unzip` should be double-quoted. I would also add a space before the `>` operator on that line:

    unzip "apache-maven-$1-bin.zip" -d ~/maven > /dev/null